docs/builds/guides/frameworks/react.md

```diff
@@ -139,6 +139,20 @@ class App extends Component {
 export default App;
 ```
 
+### Note: Using the editor with collaboration plugins
```

Comment:
```suggestion
### Using the editor with collaboration plugins
```

Reply:
🤔 I agree that `Note:` should not be a part of the heading. But the two headings above contain the `Note:` part as well 😅

Reply:
Good point, let's remove the other two, then? ;)

Reply:
I've removed them. Please, take a look now :)